pkg/client/client.go

```diff
@@ -1585,6 +1600,11 @@ func (c *Client) CreateOrUpdatePodDisruptionBudget(ctx context.Context, pdb *pol
 	return err
 }
 
+func (c *Client) CreateOrUpdateNetworkPolicy(ctx context.Context, netpol *networkingv1.NetworkPolicy) error {
+	_, _, err := resourceapply.ApplyNetworkPolicy(ctx, c.kclient.NetworkingV1(), c.eventRecorder, netpol)
```

Comment:
(non-blocking, could be addressed as a follow-up)

FYI there is a recent hotfix landed in `ApplyNetworkPolicy` utils function to address few reconciliation flaws: https://github.com/openshift/library-go/pull/2042. This patch has been cherrypicked onto `release-4.20` branch as well. You might want to adapt this fix by bumping deps with latest `o/library-go`.

Reply:
thanks, will do